pipeline-modules/common/aws/pipeline-config.yaml

```diff
@@ -157,7 +159,34 @@ template: |
       phases:
         pre_build:
           commands:
-            - echo "Cloning the pipeline modules..."
+            # installing jq v1.6
```

Comment:
I don't think this is required. see like 193. it's calling install-jq.sh. and the env file generation is also moved to scripts I think

Reply:
Update as suggested